test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -184,16 +190,20 @@ contract('KnownRelaysManager 2', function (accounts) {
       penalizer = await Penalizer.new()
       relayHub = await deployHub(stakeManager.address, penalizer.address)
       config = configureGSN({
```

Comment:
lost you here. you have both delpoyment and config, and it seems you don't use the config.

Reply:
Config only has a 'prefferedRelays' set. I did not get too deep into the code as this is simply a refactor.


See here: 
KnownRelaysManager.test.ts:208:7
knownRelaysManager = new KnownRelaysManager(contractInteractor, logger, **config**)